internal/services/search/search_service_resource.go

```diff
@@ -281,6 +298,7 @@ func resourceSearchServiceCreate(d *pluginsdk.ResourceData, meta interface{}) er
 			DisableLocalAuth: pointer.To(!localAuthenticationEnabled),
 			PartitionCount:   pointer.To(partitionCount),
 			ReplicaCount:     pointer.To(replicaCount),
+			SemanticSearch:   pointer.To(services.SearchSemanticSearch(semanticSearchSku)),
```

Comment:
in this case can we conditionally assign this field depending on whether it's `!= ""`

Reply:
Fixed.